Sources/AuthFoundation/JWT/Extensions/ClaimConvertable+Extensions.swift

```diff
@@ -12,25 +12,42 @@
 
 import Foundation
 
+@_documentation(visibility: private)
 extension String: ClaimConvertable {}
+
+@_documentation(visibility: private)
 extension Bool: ClaimConvertable {}
+
+@_documentation(visibility: private)
 extension Int: ClaimConvertable {}
+
+@_documentation(visibility: private)
 extension Double: ClaimConvertable {}
+
+@_documentation(visibility: private)
 extension Float: ClaimConvertable {}
-extension Array<String>: ClaimConvertable {}
-extension Dictionary<String, String>: ClaimConvertable {}
```

Comment:
Does JWTClaim take care of array and dictionary?

Reply:
It's unnecessary since the accessors for working with claims (e.g. `value(...)` or `subscript` methods) address arrays or dictionaries already.

The goal for ClaimConvertable is to map some JSON primitive supplied from the server into a more developer-friendly value (e.g. from a string to `URL`, or to an enum, etc.). As a result, arrays and dictionaries themselves aren't all that interesting to be converted.